test-data/unit/check-classes.test

```diff
@@ -122,7 +122,7 @@ A().f = None # E: Cannot assign to a method
 [case testReferToInvalidAttribute]
 
 class A:
-    def __init__(self):
+    def __init__(self) -> None:
```

Comment:
I think that omitting the return type was intentional here. It would be okay to have two variants of this method, one with an annotation for `__init__` and another without.